spec/cob_index/macros/custom_spec.rb

```diff
@@ -344,14 +344,15 @@
       end
     end
 
-    context "Tag 730 contains i subfield" do
-      it "extracts uniform title fields display with subfield i expected way" do
-        expected = { "title_uniform_display" => [JSON.dump({ "relation" => "Container of (work):", "title" => "Scar of shame (Motion picture)" })] }
-        expect(subject.map_record(records[4])).to eq(expected)
-      end
-    end
-
-    context "Tag 130 and not 730" do
+  # This method no longer includes tag 730
```

Comment:
@ebtoner Would it be better to remove the test since it no longer applies?

Reply:
@sensei100 -- I went back and forth about that. I decided to keep it in comments since the extract_uniform_title was built with 730 in mind, so I wanted to preserve some documentation of that.

Reply:
You've convinced me :) See the updates in the latest commit.